ui/src/singletons/store.ts

```diff
@@ -233,7 +244,39 @@ const useStore = create<AppStore>()(
             }
           }),
       },
-    }),
+    }), {
+
+      limit: 50,
+
+        handleSet: (handleSet) =>
+          throttle<typeof handleSet>(1500, (state) => {
```

Comment:
:question: Would we be able to use the [debounce function in utils](https://github.com/OctoConsulting/keip-canvas/blob/main/ui/src/utils/debounce.ts) for this? It would be preferable not to pull in more libraries prematurely.

Reply:
I have tried to implement it, but I am running into some issues. The first is the actual reactflow element used in flowCanvas has element outside of the of the React.HTML components so i cannot access those values for ex. nodes, edges.   In an example debounce function there is a callback function ev: ChangeEvent<HTMLTextAreaElement> where they are accessing the value within this HTMLTextAreaElement but this is where I am running into issues where im not to certain what I just put in the type.

Reply:
Resolved by adding a leading trigger to the `debounce` function